tests/HeaderDataTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+use Pantheon\EI\HeaderData;
+
+final class HeaderDataTest extends TestCase
+{
+  public function testConstructor(): void {
```

Comment:
I would generally advocate for adding docblocks to test functions. I often start with these to frame what I want to test and how I go about writing the tests. But they can also be functionally useful in PHPUnit, specifically, allowing you to [group](https://phpunit.readthedocs.io/en/9.5/annotations.html#group) tests together (so you can run `phpunit --group groupName`) and add [dataProvider](https://phpunit.readthedocs.io/en/9.5/writing-tests-for-phpunit.html#data-providers)s to run tests on different data sets (more on data providers here: https://tighten.co/blog/tidying-up-your-phpunit-tests-with-data-providers/ -- I tried to find examples of data providers I've written, but I'm pretty sure they're all in private repos 😞).

This isn't a blocker to this PR, though.

Reply:
Added the Groups, but had troubles getting the dataProvider to work. Well, got it to work, but needed a bunch of conditionals to achieve the same test coverage. Let's roll with this, and consider a move later on.